src/components/breadcrumb/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import classnames from 'classnames'
+
+export interface BreadcrumbProps {
+  label: string
+  href?: string
+  current?: boolean
+  withMetaData?: boolean
+  position?: number
+}
+
+export const Breadcrumb = (
+  props: BreadcrumbProps & JSX.IntrinsicElements['li']
+): React.ReactElement => {
+  const {
+    label,
+    href,
+    current = false,
+    withMetaData = false,
+    position = undefined,
+    className,
+    ...listItemProps
+  } = props
+
+  const assignMetaDataValue = (value: string): string | undefined => {
+    return withMetaData ? value : undefined
+  }
+
+  const metaData = {
+    li: {
+      property: assignMetaDataValue('itemListElement'),
+      typeof: assignMetaDataValue('ListItem'),
+    },
+    a: {
+      property: assignMetaDataValue('item'),
+      typeof: assignMetaDataValue('WebPage'),
+    },
+    span: {
+      property: assignMetaDataValue('name'),
+    },
+  }
```

Comment:
The main uncertainty I have is regarding meta data as a whole. The current implementation hardcodes the metadata as shown by uswds [here](https://github.com/uswds/uswds/blob/develop/src/components/breadcrumb/_breadcrumb--metadata.njk)

In #337 we have the following note: 

> withMetadata adds markup SEO support, includes <meta> tags and adds special attributes to the ol. Make this prop a boolean and only support JSON-LD for first go around.

I'm reading up on JSON-LD and RDFa now, but I suspect the implementation for metaData will look a bit different than it currently does.

Reply:
This was particularly helpful when learning about RDFA vs JSON-LD meta data: https://developers.google.com/search/docs/data-types/breadcrumb#json-ld_1

It seems that JSON-LD meta data should be built and included at the root level of an application, so I'm hesitant to attach it to this component. I would imagine a developer will use a specific JSON-LD npm module to include meta data in their site if they choose to go that way. In the name of keeping this repo lightweight with a focus on presentational components, I recommend leaving JSON-LD implementation to implementing teams.

We still include RDFA meta data tags here, though, since they are attached directly to the Breadcrumbs, and we'll be mirroring the uswds project by doing so (Their implementation linked in the above comment for reference)